_posts/guides/2021-12-21-guide_gamemode-info-parkour.md

```diff
@@ -24,9 +24,11 @@ Momentum Mod's implementation of Parkour comes from [the implementation found in
 
 ## History of Parkour
 
-Titanfall 2 was released, and boom, Parkour.
+Parkour, an athletic pursuit originating in France in the 20th century. Originally it began as a form of athletic training for soldiers called the "méthode naturelle" (natural method). These exercises were very effective at training balance, strength, and agility. The courses they would run were known as "parcours du combant", from which the word "parkour" originates. This information has nothing to do with video games, however, and is therefore worthless.
 
-TODO fill this out
+The first instance of parkour actually worth caring about happened in 2007 when Assasin's Creed was released, and then Mirror's Edge the next year. Those early attempts laid the groundwork. EA then applied what they had learned developing Mirror's Edge developing Titanfall 2, and subsequently showed everyone else how a parkour system was *meant* to feel. It wed the ideas of extensive first person parkour systems with a newfound fluidity and expressiveness that was miles ahead of the slow, precise, clunky systems of the past.
```

Comment:
Mirror's Edge was developed by DICE and published by EA, Titanfall 1 and 2 was developed by Respawn and published by EA as well. EA doesn't do development.